ui/apps/platform/src/Components/Select.test.jsx

```diff
@@ -23,11 +23,7 @@ describe('Component:Select', () => {
     it('should show the placeholder as the first option', () => {
         // arrange
         render(
-            <Select
-                options={initialOptions}
-                placeholder={initialPlaceholder}
-                onChange={jest.fn()}
-            />
+            <Select options={initialOptions} placeholder={initialPlaceholder} onChange={vi.fn()} />
```

Comment:
This reminds me that Jasmine decision to inject globals casts a long shadow.

For Node.js `require` or ECMAScript ` import` then we could have renamed once per file?